Doc/library/locale.rst

```diff
@@ -454,11 +454,16 @@ The :mod:`locale` module defines the following exception and functions:
 
 .. data:: LC_CTYPE
 
-   .. index:: pair: module; string
-
-   Locale category for the character type functions.  Depending on the settings of
-   this category, the functions of module :mod:`string` dealing with case change
-   their behaviour.
+   Locale category for the character type functions.  Most importantly, this
+   category defines the text encoding, i.e. how bytes are interpreted as
+   Unicode codepoints.  See :pep:`538` and :pep:`540` for how this variable
+   might be automatically coerced to ``C.UTF-8`` to avoid issues created by
+   invalid settings in containers or incompatible settings passed over remote
+   SSH connections.
+
+   Python doesn't internally use locale-dependent character transformation functions
+   from ``ctype.h``. Instead, an internal ``pyctype.h`` provides locale-independent
+   equivalents like :c:macro:`Py_TOLOWER`.
```

Comment:
This will suppress the docs warning:
```suggestion
   equivalents like :c:macro:`!Py_TOLOWER`.
```

Ideally we'd document the macro, but we're also ignoring it in Doc/whatsnew/3.13.rst (where it's ``` :c:func:`!Py_TOLOWER` ``` rather than `:c:macro:`)